frontend/www/js/omegaup/components/notification/Clarifications.vue

```diff
@@ -12,8 +12,7 @@
       <span
         class="notification-counter label"
         v-bind:class="{ 'label-danger': unread }"
```

Comment:
```suggestion
        v-bind:class="{ 'label-danger': clarifications.length > 0 }"
```

esto funciona? si sí, puedes eliminar `unread` abajo!

Reply:
Listo, si funcionó.